CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 - Add screenshot at crash (#1920)
 
+### Fixes
+
+- Remove WebKit optimization check (#1921)
```

Comment:
`m`: I think something is not right with the changelog.

Reply:
Why? LGTM